data/sql/updates/pending_db_world/Remove-AQ40-idols-from-AQ20.sql

```diff
@@ -0,0 +1,24 @@
+DELETE FROM creature_loot_template
+WHERE (
+Item = 20874 #Idol of the Sun
+OR Item = 20875 #Idol of Night
+OR Item = 20876 #Idol of Death
+OR Item = 20877 #Idol of the Sage
+OR Item = 20878 #Idol of Rebirth
+OR Item = 20879 #Idol of Life
+OR Item = 20881 #Idol of Strife
+OR Item = 20882 #Idol of War
```

Comment:
Use IN instead of this, same below

Reply:
Good to know, thank you, I changed it :D